src/app/validators/custom-validators.ts

```diff
@@ -112,7 +100,7 @@ export class CustomValidators {
       }
 
       // if start date has not been given a value yet return back
-      if (!startDate) {
+      if (!startDate || !endDate.value) {
```

Comment:
This should do to allow the end date to be blank after it has been given a value.

Reply:
@lmmrssa